Source/Core/Core/HW/WiimoteCommon/WiimoteConstants.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+// Wiimote internal codes
+
+// Communication channels
+namespace WiimoteChannel
+{
+constexpr u8 const OUTPUT_CHANNEL = 0x11;
```

Comment:
It's redundant to specify value constness on a constexpr identifier (constexpr implies const).